pages/marketplace.js

```diff
@@ -0,0 +1,13 @@
+import StarshipList from "@/components/StarshipList";
+import AppTitle from "@/components/AppTitle";
+import MarketplaceTitle from "@/components/MarketplaceTitle";
+
+export default function Marketplace() {
+  return (
+    <>
+      <AppTitle />
+      <MarketplaceTitle />
```

Comment:
Since the titles are not doing anything particularly complicated, you can keep the `<h1>` and `<h2>` here and save yourself some compos.

Also if you don't plan to style your headings much differently across your app...you can still use the global stylesheet! #oldschoolCSS